tox.ini

```diff
@@ -119,6 +119,10 @@ envlist =
     py3{7,8,9,10}-test-instrumentation-pymongo
     pypy3-test-instrumentation-pymongo
 
+    ; opentelemetry-instrumentation-pymssql
+    py3{7,8,9,10}-test-instrumentation-pymssql
+    ; instrumentation-pymssql intentionally excluded from pypy3
```

Comment:
pymssql has no pypy support:

https://github.com/pymssql/pymssql/pull/517

Reply:
If you could add that link in the file as a comment that'd be good.

Reply:
Please link this in the tox file.